pkg/fakerp/server.go

```diff
@@ -101,6 +101,19 @@ func (s *Server) validate(w http.ResponseWriter, r *http.Request) bool {
 }
 
 func (s *Server) handleDelete(w http.ResponseWriter, req *http.Request) {
+
+	config := &api.PluginConfig{
+		AcceptLanguages: []string{"en-us"},
+	}
+
+	// simulate Context with property bag
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Hour)
```

Comment:
Why the timeout change here?

Reply:
I used the same context for both objects now. We can reduce it to 30 min if there was a reason for it.

Reply:
The reason is that 1 hour for a DELETE is ridiculously long. If it starts taking longer than 30m, we need to know. Even 30m is a lot so we may want to reduce but I wouldn't do that now :)